addons/website_version/models/ir_ui_view.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+from lxml import etree
+from openerp import tools, fields, models, api
+
+
+class view(models.Model):
+
+    _inherit = "ir.ui.view"
+
+    version_id = fields.Many2one('website_version.version', ondelete='cascade', string="version_id")
+
+    @api.multi
+    def write(self, vals, toggle=False):
+        if self.env.context is None:
+            self.env.context = {}
+
+        version_id = self.env.context.get('version_id')
+        #toggle is true when changing active field
+        if version_id and not self.env.context.get('write_on_view') and not toggle:
+            self.env.context = dict(self.env.context, write_on_view=True)
+            snap = self.env['website_version.version']
+            version = snap.browse(version_id)
+            website_id = version.website_id.id
+            version_view_ids = self.env['ir.ui.view']
```

Comment:
version_view_ids: indicates this variable holds several views, linked to the current version ;) . Here I would name it View.